vm/devices/storage/disk_backend_resources/src/lib.rs

```diff
@@ -8,37 +8,29 @@
 
 use mesh::MeshPayload;
 use vm_resource::kind::DiskHandleKind;
+use vm_resource::kind::DiskLayerHandleKind;
+use vm_resource::IntoResource;
 use vm_resource::Resource;
 use vm_resource::ResourceId;
 
 // Define config types here so that you don't have to pull in the individual
 // crates just to describe the configuration.
 
-/// RAM disk handle.
+/// RAM disk layer handle.
 ///
 /// FUTURE: allocate shared memory here so that the disk can be migrated between
 /// processes.
 #[derive(MeshPayload)]
-pub struct RamDiskHandle {
```

Comment:
We could consider keeping this around as a special case.

Reply:
I'm ok with a bit of aliasing in cases of common disk configurations that just-so-happen to use LayeredDisk under-the-hood. i.e: I would probably have SqlDisk also have its own handle

Reply:
I dunno. That seems like a lot of redundancy if you're going to do it for `SqlDisk`, too.

Reply:
in that case, I'd avoid special-casing anything